libzim/wrapper.pyx

```diff
@@ -390,68 +390,72 @@ cdef class Item:
 
     @property
     def mimetype(self) -> str:
-        """ Article's mimetype -> str """
         return self.c_item.getMimetype().decode('UTF-8')
 
     def __repr__(self):
-        return f"{self.__class__.__name__}(url={self.longurl}, title={self.title})"
+        return f"{self.__class__.__name__}(url={self.path}, title={self.title})"
 
 
 
 
 #########################
-#        File           #
+#        Archive        #
 #########################
 
 cdef class PyArchive:
-    """ Zim File Reader
+    """ Zim Archive Reader
 
         Attributes
         ----------
         *c_archive : Archive
             a pointer to a C++ Archive object
         _filename : pathlib.Path
-            the file name of the File Reader object """
+            the file name of the Archive Reader object """
 
     cdef wrapper.ZimArchive* c_archive
     cdef object _filename
 
     def __cinit__(self, object filename: pathlib.Path):
-        """ Constructs a File from full zim file path
+        """ Constructs an Archive from full zim file path
 
             Parameters
             ----------
             filename : pathlib.Path
                 Full path to a zim file """
 
         self.c_archive = new wrapper.ZimArchive(str(filename).encode('UTF-8'))
-        self._filename = pathlib.Path(self.c_archive.getFilename().decode("UTF-8", "strict")) 
+        self._filename = pathlib.Path(self.c_archive.getFilename().decode("UTF-8", "strict"))
 
     def __dealloc__(self):
         if self.c_archive != NULL:
             del self.c_archive
 
+    def __enter__(self):
+        return self
+
+    def __exit__(self, *args):
+        pass
+
```

Comment:
I'm not sure we want this.
The context manager implies a under some action are valid. When the context exit, you cannot do some stuff.

This is needed when creating a zim file (we have a context of creation, once it is ended, we cannot add more item).
This is useful with "plain" file (where we want to close the file at the end of the context)

But here, the context represent "nothing". You can open a archive, get a item, close the archive and still use the item after.
And we do nothing when we enter or close the context.